packages/graphback-cli/src/helpers/db.ts

```diff
@@ -36,21 +45,18 @@ export const createDBResources = async(): Promise<void> => {
       process.exit(0)
     }
 
-    const configPath = `${process.cwd()}/config.json`
-
-    const file = readFileSync(configPath, "utf8");
-    let genConfig = {};
-    if (file) {
-      genConfig = JSON.parse(file).generation
+    const { database, dbConfig, generation } = JSON.parse(readFileSync(configPath, "utf8"))
+    
+    if(database === 'sqlite3') {
+      await execa('touch', ['db.sqlite'])
```

Comment:
Creating empty file seems strange. Maybe driver has option to created it on conenction

Reply:
Didn't find any option in `knex`